nurseconnect/services.py

```diff
@@ -25,10 +27,13 @@ def get_clinic_code(clinic_code):
             logger.error("JSON Error: {}".format(e))
             return None
 
-        if data and ("rows" in data):
-            for clinic in data["rows"]:
-                if clinic_code == clinic[0]:
-                    return clinic
+        if data and ("rows" in data) and len(data["rows"]) >= 1:
+                return data["rows"][0]
+        else:
+            logger.error(
+                "Returned data in unexpected format: {}".format(
+                    data if data is not None else "None"))
+            return None
```

Comment:
I think this might log an error even for a valid case. If the clinic code entered isn't valid, then `len(data["rows"])` will be `0`, which isn't an error in the format returned, just that the user entered the incorrect clinic code.